exercises/concept/meltdown-mitigation/conditionals_test.py

```diff
@@ -32,7 +32,7 @@ def test_is_criticality_balanced(self):
                 actual_result = is_criticality_balanced(temp, neutrons_emitted)
                 failure_message = (f'Expected {expected} but returned {actual_result} '
                                    f'with T={temp} and neutrinos={neutrons_emitted}')
-                self.assertEqual(actual_result, expected, failure_message)
+                self.assertEqual(expected, actual_result, failure_message)
```

Comment:
I am not a python testing expert, but I would expect that actual comes first and then expected, and then the failure message (optional).

Are you sure this correction is a correction?

Reply:
@kotp -- the documentation for the latest Python version (3.10.2) of [`unittest.TestCase`](https://docs.python.org/3/library/unittest.html?highlight=unittest#basic-example) also shows  `AssertEqual(actual, expected, msg)` -- as does the [quick example](https://docs.pytest.org/en/7.0.x/#a-quick-example) in the Pytest documentation with `assert actual == expected`.